styles.css

```diff
@@ -2,4 +2,41 @@ body {
     background-color: lightseagreen;
 }
 
+.demo{
+    display: flex;
+}
+
+.demo-image img{
+    height: auto;
+    width: 55em;
+    display: inline-block;
+}
+
+.example-circles{
```

Comment:
Let's add some specificity here for everything inside the `.demo` section. 

So this for example would be `.demo .example-circles {`